src/constants/MarkdownFiles/posts/2025-06-21-dmp-25-AmanNaik-week03.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "DMP ’25 Week 3 Update by Aman Naik"
+excerpt: "This week's focus was developing a working demo for the Story Builder feature using Streamlit and gathering mentor feedback for further improvements."
+category: "DEVELOPER NEWS"
+date: "2025-06-21"
+slug: "2025-06-21-dmp-25-AmanNaik-week03"
+author: "@/constants/MarkdownFiles/authors/amannaik247.md"
+tags: "dmp25,writeactivity,write,sugarlabs,week03,amannaik247"
+image: "assets/Images/c4gt_DMP.png"
+---
+
+<!-- markdownlint-disable -->
+
+# Week 3 Progress Report by Aman Naik
+
+**Project:** [Add an AI-assistant to the Write Activity](https://github.com/sugarlabs/write-activity/issues/52)  
+**Mentors:** [Walter Bender](https://github.com/walterbender), [Ibiam Chihurumnaya](https://github.com/chimosky)  
+**Reporting Period:** 2025-06-16 – 2025-06-21  
+
+---
+
+## Goals for This Week
+
+- **Goal 1:** Develop a working demo of the Story Builder feature  
+- **Goal 2:** Implement the demo using Streamlit  
+- **Goal 3:** Gather feedback from mentors and plan improvements  
+
+---
+
+## This Week’s Achievements
+
+1. **Developed a Demo of the Story Builder Feature**  
+   - Created a functional web application using Streamlit to demonstrate how the story builder AI guides students through storytelling. The demo mimics the question-and-answer-based flow and simulates an AI companion.  
+   - Find the demo [here](https://story-builder-ai.streamlit.app/)
```

Comment:
I think that somewhere here, you should have a screenshot of the app.

I imagine that the app may not always work in the future, so having at least a screenshot would help what you write here remain more meaningful longer.

Here's a sample you could use (but feel free to make your own -- this one prob doesn't have a good aspect ratio):

![Screenshot from 2025-06-22 21-17-32](https://github.com/user-attachments/assets/6ae22bc3-8d4f-4116-97af-f34fba7e0978)